clients/drcachesim/tools/schedule_stats.cpp

```diff
@@ -508,6 +518,10 @@ schedule_stats_t::aggregate_results(counters_t &total)
     // will be 0; for mock streams in tests it will be < 0; otherwise, the scheduler
     // may see more migrations due to inputs not yet executed moving among runqueues.
     assert(total.migrations <= 0. || total.migrations >= total.observed_migrations);
+
+    for (const auto &entry : cpu_footprint) {
```

Comment:
Do we need this? Isn't cores_per_thread indeed already updated in `counters_t::operator+=`?

Reply:
cores_per_thread is not updated incrementally so the += is adding empty histograms: we don't have any data until the aggregation when we compute the data from the counters.threads sets.  Added a comment here (there's already one in the +=).